catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -265,6 +272,8 @@ private void addMetacardDates(final XstreamPathValueTracker pathValueTracker,
         metacard.setModifiedDate(date);
         metacard.setCreatedDate(date);
         metacard.setEffectiveDate(date);
+        metacard.setAttribute(Core.METACARD_MODIFIED, date);
```

Comment:
I'm not sure about all these dates. The core set is supposed to describe the resource, and the metacard set is supposed to describe the metacard.

If the metacard fields are not set, there is a PR up to set them in the `StandardMetacardGroomerPlugin`, so I don't think it makes sense to set them here.

If there is no good mapping for a core/resource date, I think it is appropriate to just leave it unset.

Then again, looking at the value of the `*_STAMP_PATH` fields, it looks like that date is for the metadata and not the resource, so maybe it should map to metacard instead.